app/settings.py

```diff
@@ -59,8 +59,11 @@ def utcoffset_or_fail(date_value, key):
 
 EQ_PUBLISHER_BACKEND = os.getenv("EQ_PUBLISHER_BACKEND")
 EQ_FULFILMENT_TOPIC_ID = os.getenv("EQ_FULFILMENT_TOPIC_ID", "eq-fulfilment-topic")
-EQ_SUBMISSION_CONFIRMATION_TOPIC_ID = os.getenv(
-    "EQ_SUBMISSION_CONFIRMATION_TOPIC_ID", "eq-submission-confirmation-topic"
+EQ_SUBMISSION_CONFIRMATION_QUEUE = os.getenv(
+    "EQ_SUBMISSION_CONFIRMATION_QUEUE", "eq-submission-confirmation"
+)
+EQ_SUBMISSION_CONFIRMATION_BACKEND = os.getenv(
+    "EQ_SUBMISSION_CONFIRMATION_BACKEND", "cloud-tasks"
```

Comment:
The default should be removed to enforce that it has to be set explicitly. The validation should check the value instead of defaulting to log publisher to be consistent with other backends.